test/test_xr_kernel.py

```diff
@@ -15,23 +15,11 @@ class KernelTests(jupyter_kernel_test.KernelTests):
     kernel_name = "xr"
     language_name = "R"
 
-    def _execute_code(self, code, tests=True, silent=False, store_history=True):
-        self.flush_channels()
-
-        reply, output_msgs = self.execute_helper(code, silent=silent, store_history=store_history)
-
-        self.assertEqual(reply['content']['status'], 'ok', '{0}: {0}'.format(reply['content'].get('ename'), reply['content'].get('evalue')))
-        if tests:
-            self.assertGreaterEqual(len(output_msgs), 1)
-            # xeusr does the same as irkernel: only sends display_data, not execute_result
-            self.assertEqual(output_msgs[0]['msg_type'], 'display_data')
-        return reply, output_msgs
-
     code_hello_world = "cat('hello, world')"
     # code_page_something = "?cat"
     completion_samples = [{"text": "H", "matches": {"Hello", "Hey", "Howdy"}}]
     #complete_code_samples = ["hello, world"]
-    code_execute_result = [{"code": "6*7", "result": "[1] 42"}]
+    code_execute_result = [{"code": "6*7", "result": ["[1] 42"]}]
```

Comment:
This essentially removes the IRkernel's hack of making everything go through `display_data()` when this really is an execution result. 

So the default test using `code_execute_result` works here.